e2e/awslogs_test.go

```diff
@@ -47,12 +59,193 @@ var testAwslogs = func() {
 				config.WithEndpointResolverWithOptions(customResolver))
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 			cwClient = cloudwatchlogs.NewFromConfig(cfg)
-			cleanupAwslogs(cwClient, testAwslogsGroup, testAwslogsStream)
+			deleteLogGroup(cwClient, testAwslogsGroup)
+			deleteLogGroup(cwClient, nonExistentAwslogsGroup)
```

Comment:
why delete before and after? In case of some failure to delete from a previous run?

Reply:
Yes. Also want to take best effort to make the test environment clean.